src/Caching/Impl/Microsoft.Python.Analysis.Caching.csproj

```diff
@@ -25,7 +25,7 @@
     <Compile Remove="Models\GenericParameterModel.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="LiteDB" Version="4.1.4" />
+    <PackageReference Include="LiteDB" Version="5.0.8" />
```

Comment:
Are all of these upgrades required, or just the addition of arm64? This LiteDB upgrade contains breaking format changes that will break the existing code.